test/unittests/mongoBackend/mongoUpdateContext_test.cpp

```diff
@@ -9199,110 +9199,6 @@ TEST(mongoUpdateContextRequest, servicePathEntityUpdate_3levels)
 
 /* ****************************************************************************
 *
-* servicePathEntityUpdates_2levels -
-*
-*/
-TEST(mongoUpdateContextRequest, servicePathEntityUpdate_2levels)
```

Comment:
I think you told me but I forget... :$

Why these 3 unit tests has been removed? Looking to the removed code, I don't see the situation they are modeling is impossible (from a cliente API perpective)... maybe they have been removed because they are redundant with other existing tests?